ailab/db/finesse/test_queries/__init__.py

```diff
@@ -44,3 +44,39 @@ def get_random_chunk(cursor, schema_version, seed=None):
 
     cursor.execute(query)
     return cursor.fetchall()
+
+
+def get_random_document_score(cursor, schema_version, seed=None):
+    if seed is None:
+        seed = math.sin(time.time())
+
+    # Execute the SET commands separately
+    cursor.execute(f'SET SEARCH_PATH TO "{schema_version}", public;')
+    cursor.execute(f"SET SEED TO {seed};")
+
+    query = """
+        WITH random_crawl AS (  
+            SELECT id  
+            FROM crawl  
+            ORDER BY
```

Comment:
this came up before; ordering randomly the table is expensive means you are fetching every row and then picking up one. 

it's cheaper to pick a row between 1 and count(*) and use that with OFFSET combined with LIMIT:

https://www.postgresql.org/docs/current/queries-limit.html

Reply:
Yes you told me about that, when I copy paste the query I forgot to change `ORDER` by `OFFSET`. It will be changed soon.